action/alarmWebAction.js

```diff
@@ -103,6 +103,7 @@ function main(params) {
                     }
                     newTrigger.cron = params.cron;
                     newTrigger.timezone = params.timezone;
+                    newTrigger.strict = params.strict !== false;
```

Comment:
We need to update the document too.

Reply:
Can we make the default of this value configurable?
So in some deployment, this feature will be disabled by default.